docs/source/api/stats/xgi.stats.EdgeStat.rst

```diff
@@ -0,0 +1,35 @@
+﻿xgi.stats.EdgeStat
```

Comment:
There seem to be a few duplicate file names here

Reply:
All of `nodestats`, `NodeStat`, and `nodestat` are valid names implemented in this PR: the module, the class, and the decorator. All three names are imported into the `xgi.stats` package, so sphinx creates all three files.

I think it's fine, at least documentation-wise, though these three names are too similar to each other that they may eventually cause a problem with end-users?

Reply:
Open to suggestions here